src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -420,6 +420,11 @@ private StringBuilder BuildBreakInsert(string condition)
             return await _debugger.CmdAsync(cmd.ToString(), resultClass);
         }
 
+        public virtual Task<Results> BreakWatch(string address, uint size, ResultClass resultClass = ResultClass.done)
+        {
+            throw new NotImplementedException();
```

Comment:
that's exactly what I was looking for. Got confused by GitHub only showing partial diffs